packages/unified/src/utilities/getParsingDependencyDicts.ts

```diff
@@ -123,7 +123,7 @@ function getNoteDependencies(ast: Node<Data>): DNodeCompositeKey[] {
 
   visit(ast, [DendronASTTypes.HASHTAG], (hashtag: HashTag, _index) => {
     renderDependencies.push({
-      fname: hashtag.value,
+      fname: hashtag.fname,
     });
   });
 
```

Comment:
@jonathanyeung shouldn't the `USERTAG` "visit" also use `fname` instead of `value`. I mean that yes we can be sure `value` must match the `fname` value but we could also refactor this to directly check for `fname` if that is anyway what is what we are looking for.

Reply:
Could be - not 100% sure.  One issue is that I don't think we're consistent in how we're using the fields across each AST Type (if I recall, what value gets stored in which property is a bit different across Hashtag/UserTag/Wikilink/etc.).  

Can you check 'usertag' under the debugger to see if it should be `usertag.value` or `usertag.fname`?

Reply:
The value at https://github.com/dendronhq/dendron/blob/0db6efdbe7affa63a7a708303f2470b7dab9832e/packages/unified/src/utilities/getParsingDependencyDicts.ts#L132 is `"@example.username"` meaning that we also have an issue with usertags and this should also point to `fname` instead.

Reply:
![image](https://user-images.githubusercontent.com/328836/197760877-be62cfca-c80d-462a-82bb-e43f31948bcb.png)
Yes so this issue is also appearing with usertags

Reply:
fix for usertag: https://github.com/dendronhq/dendron/pull/3708/commits/3594eeaed3b0508967f15f9dfbab9fa0618f6cf6